recipes/spektral/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "spektral" %}
+{% set version = "0.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spektral-{{ version }}.tar.gz
+  sha256: b08284d5227f62f327579cef526971aec7fcd4cb0c153bc031694bc4376f3a0a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - joblib
+    - lxml
+    - networkx
+    - numpy
+    - pandas
+    - python >=3.6
+    - requests
+    - scikit-learn
+    - scipy
+    - tensorflow >=2.1.0
```

Comment:
This fails because conda-forge currently only offers tensorflow 1.14, please see https://github.com/conda-forge/tensorflow-feedstock/issues